docs/download_files.md

```diff
@@ -92,31 +92,52 @@ Every download also includes the individual [QC report](#qc-report) and, if appl
 
 ## Custom datasets
 
-<!--TODO: Fill in this section with information about creating and downloading custom datasets-->
+To download data for individual samples, multiple projects, or any combination of individual samples and projects, use the `Add to Dataset` button. 
```

Comment:
I think we actually need to say something about where the selection happens int he first place to be able to "add to dataset." We have both the `Add to Dataset` button on the project card, and you can select from the samples table on the project page.

Reply:
> I think we actually need to say something about where the selection happens int he first place to be able to "add to dataset." We have both the `Add to Dataset` button on the project card, and you can select from the samples table on the project page.

I feel like this is self explantory with this sentence that's in here? You can use the button to either add a project or individual samples in various places depending what page you are on. It also moves depending on exactly what view you are looking at.

Reply:
> You can use the button to either add a project or individual samples in various places depending what page you are on

This is more or less the level of detail I'd want to see, but if you think it's overkill then let's let @dvenprasad weigh in when she reviews and tell us the right answer 😄